fenicsprecice/fenicsprecice.py

```diff
@@ -174,7 +174,9 @@ def get_point_sources(self, data):
 
         assert (self._size == 1), "get_point_sources function only works in serial."
 
-        return get_forces_as_point_sources(self._Dirichlet_Boundary, self._read_function_space, data)
+        point_data = data.copy()
+
```

Comment:
Is this required?

Reply:
This should be done to prevent the user from directly getting access to an internal dictionary object. However this has no effect on the current tutorial cases

Reply:
But in this case the user already has access to `data`, since the user provides it. I don't see a point in copying it here. Or am I missing something?

Reply:
I saw you already reverted it in df9ec95. Then this is resolved from my point of view.